.gitattributes

```diff
@@ -1,3 +1,3 @@
-* text=auto
+* text eol=lf
```

Comment:
This doesn't work like you think. It's super flawed and really means "convert **everything** to text and with `lf` newlines" instead of meaning "convert text to use `lf` newlines". So it will ruin anything binary.